docs/dxp/7.x/en/installation-and-upgrades/setting-up-liferay-dxp/clustering-for-high-availability/example-creating-a-dxp-cluster.md

```diff
@@ -163,25 +228,35 @@ You're ready to configure the DXP server nodes.
 
 1. Copy the configuration to your other node(s).
 
+    Create a `/osgi/config/` directory for the second DXP node.
```

Comment:
@jrhoun - Shouldn't it be _**an** /osgi/config/ **folder**_?

Reply:
I guess so b/c of the pronunciation of 'O-S-G-I'?

Reply:
Right :-D